backend/app/interactem/app/api/routes/operators.py

```diff
@@ -0,0 +1,28 @@
+import uuid
+from typing import Any
+
+from fastapi import APIRouter, HTTPException
+
+from interactem.app.api.deps import CurrentUser, SessionDep
+
+from interactem.app.models import Operators
+from interactem.app.operators import fetch_operators
+from interactem.core.logger import get_logger
+
+logger = get_logger()
+router = APIRouter()
+
+_operators = None
+
+
+@router.get("/", response_model=Operators)
+async def read_operators(current_user: CurrentUser) -> Operators:
+    """
+    Retrieve available operators.
+    """
+    global _operators
```

Comment:
when do we refetch them? just on api startup?

Reply:
I guess this is to go with your point about a slow first call to this endpoint...

Reply:
perhaps in future we can have a task that does this every X mins

Reply:
Yes, for now its only done on startup, in the future we might have endpoint to initial an update.